src/ui/RealmBrowser/index.tsx

```diff
@@ -342,7 +342,7 @@ class RealmBrowserContainer
     const message = err.message || '';
     const mightBeEncrypted =
       message.includes('Not a Realm file.') ||
-      message.includes('Invalid mnemonic') ||
+      message.includes('invalid mnemonic') ||
```

Comment:
@kneth @kraenhansen What about older files that had it capitalised? That won't work anymore?
So shouldn't you just add the lower case option and not remove the uppercase option?

Reply:
The core version as Studio 14 is using, has a different error message than previous versions.